### ISSUE TITLE
IE crashes on connect

### ISSUE THREAD

**Issue by Xan-Kun** [openaudible/openaudible#281]
OA 1.6.5 on windows 7
I get a lot of "Script Error" messages when trying to open the browser. It looks like an IE component, so I was wondering, what the minimum browser version OA expects is.
Couldn't find anything in the documentation about that.
Or ist there a way to use a different browser on win?

Thanks for your time.

**Comment by openaudible** [comment]
I'm not sure how the browser works. Some errors may be OK to ignore.. but not sure what you're seeing. Feel free to post some console errors. We use the Eclipse SWT Browser, which might help with googling the errors.

**Comment by Xan-Kun** [comment]
Thank you for the answer. SWT Browser basically uses IE on Win platforms. As I never use that "browser", it is pretty stripped down on my machine and has very restrictive security settings.
Unfortunately, I am not going to fancy the idea of using IE.
It would be really nice to use something like https://github.com/maketechnology/chromium.swt in the future.

**Comment by openaudible** [comment]
Yeah, nobody likes IE. But its what is supported out of the box.  Chromium.swt looks like an interesting project.  If I  have time I may try to turn it on and test it.. But lots of work and may introduce other bugs or people with incompatible systems.  

Are you able to use openaudible as intended (get list of books) ? Audible does a lot of 3rd party website traffic that we filter out. So many errors are just informational.

**Comment by Xan-Kun** [comment]
Thanks for the answer. I use it just to manually import and convert and am totally fine with that for now. I figured it somewhat out and with the security settings in IE I couldn't even load the audible home page. Works as itended :-) As I said, I never use IE and don't actually consider it a browser (it caused to much pain to be forgiven back in the days).
I used CEF heavily in a commercial project and only tried it, because I needed the native arrays support it had and was totally happy with it. Gave me less hassle then the IE component.
But that was just a very limited use-case and of course, mileage will vary ;-)
